requests_futures/sessions.py

```diff
@@ -109,3 +109,44 @@ def close(self):
         if self._owned_executor:
             self.executor.shutdown()
 
+    def get(self, url, **kwargs):
```

Comment:
I believe these will need to include and pass along `*args` in addition to `**kwargs`. 

```python
>>> class Foo(object):
...     def bar(self, **kwargs):
...         from pprint import pprint
...         pprint(kwargs)
...
>>> Foo().bar(42, 'key')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: bar() takes exactly 1 argument (3 given)
>>> Foo().bar(i=42, j='key')
{'i': 42, 'j': 'key'}
```

Also would this just change the return type in the method docs or would it also wipe out all of the existing doc on the parent methods?

Reply:
As long as method signatures are kept the same as parent methods, *args should not make a difference at calling side imo.  

However it seems that whether the docstrings are inherited depends on the tools used, and there are no official way to declare that. Should we copy docstrings from requests.Session and change the return type only, or just leave it to the tools?